sycl/test/fpga_tests/fpga_lsu.cpp

```diff
@@ -1,7 +1,4 @@
-// TODO: Enable compilation w/o -fno-sycl-early-optimizations option.
-// See https://github.com/intel/llvm/issues/2264 for more details.
-
-// RUN: %clangxx -fsycl -fno-sycl-early-optimizations %s -o %t.out
+// RUN: %clangxx -fsycl %s -o %t.out
```

Comment:
I would add also test of this sample w/o optimizations to check that all works fine in both cases.

Reply:
Is there any reason to do this for fpga_lsu test only? 
If no, I suggest file a separate request to enable such testing for all lit tests.

Reply:
I think that this testing might be added for other lit tests as well.